config/locales/client.en.yml

```diff
@@ -1897,21 +1897,27 @@ en:
       google_oauth2:
         name: "Google"
         title: "with Google"
+        a11y_title: "Login with Google"
```

Comment:
I think I would prefer something like: sr_title or screen_reader_title

Reply:
Sure, will change this, and we should be consistent with this internally